language/move-vm/test-utils/src/gas_schedule.rs

```diff
@@ -96,7 +99,7 @@ impl GasCost {
     /// Convert a GasCost to a total gas charge in `InternalGas`.
     #[inline]
     pub fn total(&self) -> u64 {
-        self.instruction_gas.add(self.memory_gas)
+        self.instruction_gas.add(self.memory_gas) * TABLE_GAS_COST_SCALE_FACTOR
```

Comment:
This function will be called lots of times during the MoveVM runtime - maybe it's better to calculate this before the `GasCost` is even created with `new()`:
```rust
        ...
        (WriteRef, GasCost::new(735 * TABLE_GAS_COST_SCALE_FACTOR, 1)),
        (Mul, GasCost::new(588 * TABLE_GAS_COST_SCALE_FACTOR, 1)),
        ...
```
It's a bit repetitive, yes, but we'll get better runtime performance since the multiplication with this factor will happen only once during the INSTRUCTION_COST_TABLE initialization.

Once we finally decide on the value for TABLE_GAS_COST_SCALE_FACTOR, I don't think it will ever need to be configurable during the runtime.

Reply:
Done :+1: